docs/planning/future-releases/release-v2.2.0.md

```
@@ -1,3 +1,14 @@
+> **Superseded — this plan shipped as v2.3.0, not as the version in its title.**
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)